sdk/package.json

```diff
@@ -79,7 +79,7 @@
   },
   "scripts": {
     "clean": "rm -rf ./dist",
-    "copy-files": "cp -r ./proto/ ./dist/proto",
+    "copy-files": "cp -r ./proto/ ./dist/proto && cp index.d.ts ./dist",
```

Comment:
The `index.d.ts` file is generated during the `prepare` step, but the `compile` doesn't depend on that step.
Wouldn't make sense to do the copy in the `prepare.sh` script?

Reply:
the compile is executed after the `prepare.sh` ATM and the result is going to be overridden by `compile`.
But I'm open to change (maybe in another PR) the order/sequence of commands if you have a good proposal @janory 🙂